cmake/CeleritasUtils.cmake

```diff
@@ -184,6 +180,12 @@ function(celeritas_add_library target)
     endif()
   endif()
 
+  # Build all targets in lib/
+  set_target_properties(${_targets} PROPERTIES ${_props}
```

Comment:
We should actually have a `celeritas_rdc_set_target_properties` similar to `celeritas_target_include_directories` which you could use on line 170 and would apply (as it semantically) should the properties to all (3 in the cuda case) targets.  However, this is an effective solution and we could delay the full solution to another PR.

Reply:
It is nice to be able to reuse the `targets`  rather than having to rediscover them with each `rdc_install`, `rdc_set_target_properties`, etc. I think we could probably simplify some of the `CeleritasLibrary` logic with a `rdc_get_targets` command? Then we could use the regular cmake install/set/target commands without having to parse all the cmake commands themselves.

Reply:
Yeah, making it invisible is definitely best, but I'd almost rather "know what's going on" than worry about when to prefix all my cmake calls (and indeed which ones are available with `rdc_` prefixes).

Reply:
There is still a philosophical discussion but since we need to do some other updates to the rdc routine and that the one dealing with properties is not written yet (and likely to not be trivial), let's punt for now.